plugins/transfer_pokemon/__init__.py

```diff
@@ -0,0 +1,83 @@
+from pokemongo_bot.human_behaviour import sleep
+from pokemongo_bot.event_manager import manager
+
+
+def log(text, color="black"):
+    manager.fire("logging", text=text, color=color, prefix="Transfer")
+
+
+@manager.on("pokemon_bag_full", priority=-1000)
+def filter_pokemon(bot=None, transfer_list=None):
+    # type: (PokemonGoBot, Optional[List[Pokemon]]) -> Dict[Str, List[Pokemon]]
+
+    if transfer_list is None:
+        bot.api_wrapper.get_player().get_inventory()
+        response_dict = bot.api_wrapper.call()
+        transfer_list = response_dict['pokemon']
+
+    new_transfer_list = []
+    indexed_pokemon = dict()
+    for pokemon in transfer_list:
+
+        # ignore deployed pokemon
+        if pokemon.deployed_fort_id is not None:
+            continue
+
+        pokemon_num = pokemon.pokemon_id
+        if pokemon_num not in indexed_pokemon:
+            indexed_pokemon[pokemon_num] = list()
+        indexed_pokemon[pokemon_num].append(pokemon)
+
+    if bot.config.cp and False:
```

Comment:
No worries, i'm just updating https://github.com/OpenPoGo/OpenPoGoBot/pull/115 atm, so i'll roll it into there :)